azurelinuxagent/ga/exthandlers.py

```diff
@@ -753,7 +731,7 @@ def __handle_and_report_ext_handler_errors(ext_handler_i, error, report_op, mess
             add_event(name=name, version=handler_version, op=report_op, is_success=False, log_event=True,
                       message=message)
 
-    def __handle_ext_disallowed_error(self, ext_handler_i, error_code, report_op, message, extension):
+    def __handle_ext_disallowed_error(self, ext_handler_i, error_code, report_op, message, extension, report):
```

Comment:
Let's not keep adding parameters to those functions, it leads to multiple issues (add code complexity, make testing harder, etc). We are forced to suppress linter warnings about too many parameters because fixing the existing code is too expensive, but we should try hard not to do it on new code. I see that this argument is always True, but for one single case: signature validation. Please take a second look and see if you can avoid this extra parameter by doing some re-design, or refactoring.